homeassistant/components/doorbird.py

```diff
@@ -24,60 +26,139 @@
 CONF_DOORBELL_EVENTS = 'doorbell_events'
 CONF_CUSTOM_URL = 'hass_url_override'
 
+DOORBELL_EVENT = 'doorbell'
+MOTION_EVENT = 'motionsensor'
```

Comment:
Could you call the event motion? I will approved and merge the PR afterwards.

Reply:
That is the event name required by the device API for event subscriptions.

![image](https://user-images.githubusercontent.com/453840/41204087-b7327194-6ca5-11e8-87e2-b58a5c723f36.png)

https://www.doorbird.com/downloads/api_lan.pdf?rev=0.20

Reply:
Got it. Sounds a bit weird.

Reply:
@oblogic7 That's only for `monitor.cgi`. The new `schedule.cgi` uses 'motion'. No need to update this PR, I'm fixing it in mine :slightly_smiling_face: 

![image](https://user-images.githubusercontent.com/1693960/41257665-53b82f16-6d93-11e8-9770-5d294701150a.png)